corehq/apps/app_manager/suite_xml/sections/details.py

```diff
@@ -358,6 +362,23 @@ def get_detail_id_safe(self, module, detail_type):
         return detail_id if detail_id in self.active_details else None
 
 
+def get_nodeset_sort_elements(detail):
+    from corehq.apps.app_manager.models import SortElement
+    sort_elements = []
+    tab_spans = detail.get_tab_spans()
+    for tab in detail.get_tabs():
```

Comment:
This behaves correctly if there are multiple nodeset tabs, right? (I don't know the sorting code well.)